fast64_internal/f3d/f3d_writer.py

```diff
@@ -874,10 +875,10 @@ def __init__(
         texDimensions,
         material: bpy.types.Material,
         currentGroupIndex,
-        triList,
-        vtxList,
+        triGroup,
```

Comment:
```suggestion
        triGroup: FTriGroup,
```
when changing params here i'd like to start adding type annotations